src/adapters/tapsense.js

```diff
@@ -0,0 +1,87 @@
+//v0.0.1
+
+const bidfactory = require('../bidfactory.js');
+const bidmanager = require('../bidmanager.js');
+const adloader = require('../adloader');
+const utils = require('../utils.js');
+
+const TapSenseAdapter = function TapSenseAdapter() {
+  const version = "0.0.1";
+  const creativeSizes = [
+    "320x50"
```

Comment:
Just to be sure, to create our bidder documentation I will need to make a PR for this page with our information correct?? http://prebid.org/dev-docs/bidders.html